addons/mail/tests/test_mail_channel.py

```diff
@@ -315,6 +340,17 @@ def test_channel_info_get(self):
         same_solo_channel_info = self.env['mail.channel'].channel_get(partners_to=self.partner_employee_nomail.ids)
         self.assertEqual(same_solo_channel_info['id'], solo_channel_info['id'])
 
+    @users('employee')
+    def test_channel_info_get_should_update_last_activity_time(self):
+        # create the channel via `channel_get`
+        self.env['mail.channel'].channel_get(partners_to=self.partner_admin.ids)
+
+        # `last_activity_time` should be updated again when `channel_get` is called
+        retrive_time = datetime(2021, 1, 1, 0, 0)
+        with patch.object(fields.Datetime, 'now', lambda: retrive_time):
+            channel_info = self.env['mail.channel'].channel_get(partners_to=self.partner_admin.ids)
+        self.assertEqual(channel_info['last_activity_time'], retrive_time)
```

Comment:
Why do you call `channel_get` twice instead of once? Maybe need an explanation or an assert between the 2 cases (for initial value?)

Reply:
explanation is added